docs/user/intro.rst

```diff
@@ -38,10 +38,11 @@ But there is more.. you can also give your module a dictionary in an interactive
 
 ::
 
-    >>> from mymodule import MyModule
+    >>> from argschema import ArgSchemaParser
+    >>> from mymodule import MySchema
     >>> d = {'a':5}
-    >>> mod = MyModule(input_data=d)
-    >>> mod.run()
+    >>> mod = ArgSchemaParser(input_data=d,schema_type=MySchema)
```

Comment:
no i don't agree with that.. input_data is only to allow people to use the module from python, if you are passing parameters from json or from command line you don't want to have to do it.  I set it up so that I have default parameters that run if you just run the module as my default pattern, but I don't want to make people have to do that, or pass in empty dictionaries.  input_data is ignored if you pass input_json